### DISCUSSION TITLE
Media Upload not working

### DISCUSSION THREAD

**Discussion by chili-666** [pimcore/pimcore#14916] [Q&A / Support]
Hey there, 

I've been struggling with this for a couple of days now and couldn't find any solution at all. If I try to upload a media file to my assets, the loading bar stays forever and nothing happens. The file is uploaded, but is not shown in pimcore. In the developer console, there ist the following error:

![image](https://user-images.githubusercontent.com/45040285/231689695-5fc327ce-b563-459a-a1ea-60f5d9c52cd4.png)

This is all I have - dev.log and php.log don't show anything... 

If I try to import images from a folder on the server (upload via  ftp), I get the following error:
`{"success":false,"trace":[{"file":"\/home\/users\/uniadmin\/www\/pimcore\/vendor\/pimcore\/pimcore\/models\/Version.php","line":258,"function":"save","class":"Pimcore\\Model\\Version\\Adapter\\FileSystemVersionStorageAdapter","type":"-\u003E"},{"file":"\/home\/users\/uniadmin\/www\/pimcore\/vendor\/pimcore\/pimcore\/models\/Element\/AbstractElement.php","line":733,"function":"save","class":"Pimcore\\Model\\Version","type":"-\u003E"},{"file":"\/home\/users\/uniadmin\/www\/pimcore\/vendor\/pimcore\/pimcore\/models\/Asset.php","line":857,"function":"doSaveVersion","class":"Pimcore\\Model\\Element\\AbstractElement","type":"-\u003E"},{"file":"\/home\/users\/uniadmin\/www\/pimcore\/vendor\/pimcore\/pimcore\/models\/Asset.php","line":549,"function":"saveVersion","class":"Pimcore\\Model\\Asset","type":"-\u003E"},{"file":"\/home\/users\/uniadmin\/www\/pimcore\/vendor\/pimcore\/pimcore\/models\/Asset.php","line":359,"function":"save","class":"Pimcore\\Model\\Asset","type":"-\u003E"},{"file":"\/home\/users\/uniadmin\/www\/pimcore\/vendor\/pimcore\/pimcore\/bundles\/AdminBundle\/Controller\/Admin\/Asset\/AssetController.php","line":2479,"function":"create","class":"Pimcore\\Model\\Asset","type":"::"},{"file":"\/home\/users\/uniadmin\/www\/pimcore\/vendor\/symfony\/http-kernel\/HttpKernel.php","line":163,"function":"importServerFilesAction","class":"Pimcore\\Bundle\\AdminBundle\\Controller\\Admin\\Asset\\AssetController","type":"-\u003E"},{"file":"\/home\/users\/uniadmin\/www\/pimcore\/vendor\/symfony\/http-kernel\/HttpKernel.php","line":75,"function":"handleRaw","class":"Symfony\\Component\\HttpKernel\\HttpKernel","type":"-\u003E"},{"file":"\/home\/users\/uniadmin\/www\/pimcore\/vendor\/symfony\/http-kernel\/Kernel.php","line":202,"function":"handle","class":"Symfony\\Component\\HttpKernel\\HttpKernel","type":"-\u003E"},{"file":"\/home\/users\/uniadmin\/www\/pimcore\/public\/index.php","line":36,"function":"handle","class":"Symfony\\Component\\HttpKernel\\Kernel","type":"-\u003E"}],"traceString":"in \/home\/users\/uniadmin\/www\/pimcore\/vendor\/pimcore\/pimcore\/models\/Version\/Adapter\/FileSystemVersionStorageAdapter.php:102\n#0 \/home\/users\/uniadmin\/www\/pimcore\/vendor\/pimcore\/pimcore\/models\/Version.php(258): Pimcore\\Model\\Version\\Adapter\\FileSystemVersionStorageAdapter-\u003Esave()\n#1 \/home\/users\/uniadmin\/www\/pimcore\/vendor\/pimcore\/pimcore\/models\/Element\/AbstractElement.php(733): Pimcore\\Model\\Version-\u003Esave()\n#2 \/home\/users\/uniadmin\/www\/pimcore\/vendor\/pimcore\/pimcore\/models\/Asset.php(857): Pimcore\\Model\\Element\\AbstractElement-\u003EdoSaveVersion()\n#3 \/home\/users\/uniadmin\/www\/pimcore\/vendor\/pimcore\/pimcore\/models\/Asset.php(549): Pimcore\\Model\\Asset-\u003EsaveVersion()\n#4 \/home\/users\/uniadmin\/www\/pimcore\/vendor\/pimcore\/pimcore\/models\/Asset.php(359): Pimcore\\Model\\Asset-\u003Esave()\n#5 \/home\/users\/uniadmin\/www\/pimcore\/vendor\/pimcore\/pimcore\/bundles\/AdminBundle\/Controller\/Admin\/Asset\/AssetController.php(2479): Pimcore\\Model\\Asset::create()\n#6 \/home\/users\/uniadmin\/www\/pimcore\/vendor\/symfony\/http-kernel\/HttpKernel.php(163): Pimcore\\Bundle\\AdminBundle\\Controller\\Admin\\Asset\\AssetController-\u003EimportServerFilesAction()\n#7 \/home\/users\/uniadmin\/www\/pimcore\/vendor\/symfony\/http-kernel\/HttpKernel.php(75): Symfony\\Component\\HttpKernel\\HttpKernel-\u003EhandleRaw()\n#8 \/home\/users\/uniadmin\/www\/pimcore\/vendor\/symfony\/http-kernel\/Kernel.php(202): Symfony\\Component\\HttpKernel\\HttpKernel-\u003Ehandle()\n#9 \/home\/users\/uniadmin\/www\/pimcore\/public\/index.php(36): Symfony\\Component\\HttpKernel\\Kernel-\u003Ehandle()\n#10 {main}","message":"Attempted to call function \u0022link\u0022 from namespace \u0022Pimcore\\Model\\Version\\Adapter\u0022.\nDid you mean to call \u0022\\safe\\link\u0022?"}`

I am stumped - so does anyone have a clue, what part of my configuration ist wrong?

**Comment by chili-666** [comment]
Okay, that was dumb. There was a typo in the php-conf of my server...

**Reply by blackbender** [reply]
How you resolved that problem?
